cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -912,19 +912,191 @@ def create_image(self, name):
         raise NotImplementedError(
             'To be implemented after GCEVolumeService.')
 
+    @property
+    def _existing_target_instance(self):
```

Comment:
I feel all the properties here, even though they're intended as private, should have a docstring included to make it clear right away what's their purpose.